### PR TITLE
Add debug references

### PR DESCRIPTION
**Pull request by dudo**
Adding a few helpful commands for <https://kubernetes.io/docs/reference/kubectl/generated/kubectl_debug/>

### CODE REVIEW

**Comment by Shubham82** [comment]
Thanks, @dudo
The preview looks good to me.
https://deploy-preview-45959--kubernetes-io-main-staging.netlify.app/docs/reference/kubectl/quick-reference/#interacting-with-running-pods

/lgtm
/sig cli

File: content/en/docs/reference/kubectl/quick-reference.md [hunk 1]
```diff
@@ -371,6 +371,8 @@ kubectl port-forward my-pod 5000:6000               # Listen on port 5000 on the
 kubectl exec my-pod -- ls /                         # Run command in existing pod (1 container case)
 kubectl exec --stdin --tty my-pod -- /bin/sh        # Interactive shell access to a running pod (1 container case)
 kubectl exec my-pod -c my-container -- ls /         # Run command in existing pod (multi-container case)
+kubectl debug my-pod -it --image=busybox            # Create an interactive debugging session witin existing pod and immediately attach to it
+kubectl debug node/my-node -it --image=busybox      # Create an interactive debugging session on a node and immediately attach to it
```
Comment (sftim) [inline content/en/docs/reference/kubectl/quick-reference.md:375]: Please use `busybox:1.28` per other examples, or bump all the examples in this page to a different version of BusyBox.

**Comment by sftim** [comment]
We prefer to encourage people to use tags for the images they run, so:
/lgtm cancel

File: content/en/docs/reference/kubectl/quick-reference.md [hunk 2]
```diff
@@ -371,6 +371,8 @@ kubectl port-forward my-pod 5000:6000               # Listen on port 5000 on the
 kubectl exec my-pod -- ls /                         # Run command in existing pod (1 container case)
 kubectl exec --stdin --tty my-pod -- /bin/sh        # Interactive shell access to a running pod (1 container case)
 kubectl exec my-pod -c my-container -- ls /         # Run command in existing pod (multi-container case)
+kubectl debug my-pod -it --image=busybox:1.28       # Create an interactive debugging session witin existing pod and immediately attach to it
+kubectl debug node/my-node -it --image=busybox1.28  # Create an interactive debugging session on a node and immediately attach to it
```
[the remark this answers is not in the document]
Comment (dudo) [reply content/en/docs/reference/kubectl/quick-reference.md]: Working from my phone 😓

[the remark this answers is not in the document]
Comment (dudo) [reply content/en/docs/reference/kubectl/quick-reference.md]: I mean... it's more than a nit, it was syntactically incorrect.

**Comment by sftim** [comment]
BTW, this is a change to https://kubernetes.io/docs/reference/kubectl/quick-reference/ and not to https://kubernetes.io/docs/reference/kubectl/generated/kubectl_debug/

**Pull request merged**